sdk/synapse/synapse-accesscontrol/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@azure/synapse-accesscontrol",
+  "author": "Microsoft Corporation",
+  "description": "A generated SDK for AccessControlClient.",
+  "version": "1.0.0",
```

Comment:
shouldn't the version of these packages be `1.0.0-beta.1`?

Reply:
Good catch!